todoman/cli.py

```diff
@@ -48,6 +48,23 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    try:
+        if 'before' in val:
+            temp = val[7:]
+            ret = [True]
+        elif 'after' in val:
+            temp = val[6:]
+            ret = [False]
+        else:
+            raise click.BadParameter(
+                'The start date of the task should be in format \'before 2012-10-16\'')
```

Comment:
I'm also thinking that if we make the flag `--starts-before`, parsing might be a lot simpler, but I'm leaving this up to you.